src/Cassandra.Tests/Cassandra.Tests.csproj

```diff
@@ -58,10 +57,6 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net6' Or '$(TargetFramework)' == 'net7' Or '$(TargetFramework)' == 'net8' ">
-    <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
-    <PackageReference Include="System.Threading.Tasks.Parallel" Version="4.3.0" />
-  </ItemGroup>
```

Comment:
Why these changes, also the project reference needs to be updated to target `netstandard2.1`  when the test project is not targeting net framework

Reply:
https://github.com/datastax/csharp-driver/pull/617#discussion_r2046999702